src/ai/backend/agent/server.py

```diff
@@ -556,6 +557,39 @@ async def commit(
             "path": str(Path(subdir, filename)),
         }
 
+    @rpc_function
+    @collect_error
+    async def get_local_config(self):
+        def get_or_empty(val):
+            if val is None:
+                return ""
+            return str(val)
+
+        return {
+            "agent": {
+                "abuse-report-path": get_or_empty(
+                    self.local_config["agent"].get("abuse-report-path")
+                ),
```

Comment:
Instead of defining custom function, adding default value to `get()` function will be sufficient.
```suggestion
                "abuse-report-path": self.local_config["agent"].get("abuse-report-path", ""),
```